Solink.AddIn.Helpers/RestartableAddInGenerator.cs

```diff
@@ -37,6 +37,7 @@ public RestartableAddInGenerator(string namespaceName, FileInfo sourceAssembly,
 
         public void Generate()
         {
+            AppDomain.CurrentDomain.ReflectionOnlyAssemblyResolve += (sender, args) => Assembly.ReflectionOnlyLoad(args.Name);

```

Comment:
Thanks for the suggestion on how to test. So far my testing was through adding a class with reference to our private nuget libraries.